src/printer.ts

```diff
@@ -506,7 +510,7 @@ export class PugPrinter {
 				}
 			}
 			if (numNormalAttributes > 0) {
-				// Add leading and trailing parantheses
```

Comment:
Apologies for all these spelling mistakes. It was very late last night and I thought that my VSCode was using a spell checker for this repo 🤦